pkg/deployment/deployment.go

```diff
@@ -219,6 +219,37 @@ func Deploy(
 		return result, err
 	}
 
+	// ConfigureCephClient
+	readyCondition = dataplanev1beta1.ConfigureCephClientReadyCondition
+	readyWaitingMessage = dataplanev1beta1.ConfigureCephClientReadyWaitingMessage
+	readyMessage = dataplanev1beta1.ConfigureCephClientReadyMessage
+	readyErrorMessage = dataplanev1beta1.ConfigureCephClientErrorMessage
+	deployFunc = ConfigureCephClient
+	deployName = "ConfigureCephClient"
+	deployLabel = ConfigureCephClientLabel
+	result, err = ConditionalDeploy(
+		ctx,
+		helper,
+		obj,
+		sshKeySecret,
+		inventoryConfigMap,
+		status,
+		readyCondition,
+		readyMessage,
+		readyWaitingMessage,
+		readyErrorMessage,
+		deployFunc,
+		deployName,
+		deployLabel,
+		networkAttachments,
+		openStackAnsibleEERunnerImage,
+		ansibleTags,
+		extraMounts,
+	)
+	if err != nil || result.RequeueAfter > 0 {
+		return result, err
+	}
```

Comment:
it is easier to put this under an if,
as the flow is built here

Reply:
done.